### ISSUE TITLE
NTR: Liver biopsy: phosphorylase b kinase activity low (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9194]
New term request Liver biopsy: phosphorylase b kinase activity low (elthat_101019012445): Hepatic biopsy  UMLS: 1844431

**Comment by pnrobinson** [comment]
@MickeySegal
While drafting a definition, I was not sure if the "b" above refered to the beta subunit of PhK. If so, I am not sure if this is correct because I think the enzyme activity can only be measured for the entire enzyme.

Label:  Decreased hepatic phosphorylase kinase activity
Definition: Activity of phosphorylase kinase in liver tissue below the lower limit of normal.  Phosphorylase kinase (PhK) has a major regulatory role in the breakdown of glycogen. The enzyme PhK comprises four copies each of four subunits, encoded by PHKA1, PHKA2, PHKB, and PHKG.
Comment: Phosphorylase kinase (PhK) deficiency causing glycogen storage disease type IX (GSD IX) results from deficiency of the enzyme phosphorylase b kinase, which has a major regulatory role in the breakdown of glycogen. While liver biopsy or muscle biopsy was done routinely in previous years to measure PhK activity, the availability of molecular genetic testing has reduced the need for such invasive procedures.
PMID:21634085

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK55061/ "Phosphorylase kinase deficiency causing glycogen storage disease type IX (GSD IX) results from deficiency of the enzyme phosphorylase b kinase (PhK), an enzyme with a key regulatory role in the breakdown of glycogen. Deficiency of this enzyme, which is composed of four copies each of four subunits (α, β, γ, and δ), results in considerable clinical variability"

So it looks like the beta subunit is one of 4 components of phosphorylase b kinase.

**Comment by pnrobinson** [comment]
Yes, but the activity is measured for the (composite) enzyme, not for the subunit?

**Comment by pnrobinson** [comment]
@MickeySegal  -- I think the following is correct. Please open new term if needed

Adding new term: 
- ID: HP:6000333
- label: Reduced hepatic phosphorylase kinase activity
- parents: [Abnormal liver enzyme activity or concentration (HP:0034685)](https://hpo.jax.org/app/browse/term/HP:0034685)
- pmid: PMID:21634085
- def: Activity of phosphorylase kinase in liver tissue below the lower limit of normal. Phosphorylase kinase (PhK) has a major regulatory role in the breakdown of glycogen. The enzyme PhK comprises four copies each of four subunits, encoded by PHKA1, PHKA2, PHKB, and PHKG.
- comment: Phosphorylase kinase (PhK) deficiency causing glycogen storage disease type IX (GSD IX) results from deficiency of the enzyme phosphorylase b kinase, which has a major regulatory role in the breakdown of glycogen. While liver biopsy or muscle biopsy was done routinely in previous years to measure PhK activity, the availability of molecular genetic testing has reduced the need for such invasive procedures.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9194
- synonyms: Decreased hepatic phosphorylase kinase activity(NONE)
